mlflow/utils/import_hooks/__init__.py

```diff
@@ -247,7 +247,7 @@ def __init__(self):
 
     @synchronized(_post_import_hooks_lock)
     @synchronized(_import_error_hooks_lock)
-    def find_module(self, fullname, path=None):
+    def find_spec(self, fullname, path, target=None):
```

Comment:
@dbczumar Confirmed `tensorflow.keras` now refers to `site-packages/keras/api/_v2/keras/__init__.py`

```
from mlflow.utils.import_hooks import register_post_import_hook


register_post_import_hook(lambda x: x, "keras")

import tensorflow.keras

print(tensorflow.keras)
```

output:

```
<module 'tensorflow.keras' from '/Users/harutakakawamura/.pyenv/versions/miniconda3-4.7.12/envs/mlflow-dev-env/lib/python3.7/site-packages/keras/api/_v2/keras/__init__.py'>
```

Reply:
Woohoo! Awesome RCA, Haru! Really spectacular!